maltoolbox/language/classes_factory.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, lang_graph: LanguageGraph):
 
     def _generate_assets(self) -> None:
         """
-        Generate JSON Schema for the assets in the language specification.
+        Generate JSON Schema for asset types in language specification.
```

Comment:
The first 'the' being removed makes sense, the second one does not. :)

Reply:
fixed